### ISSUE TITLE
`formatEther(x)` != `formatUnits(x,18)` 

### ISSUE THREAD

**Issue by halo3mic** [gakonst/ethers-rs#2356]
In ethers.js `formatEther(x) == formatUnits(x, 18)`, I noticed this is not the case with ethers.rs.  `format_ether` truncates the input number and returns U256, while`format_units` returns the input number in decimal representation as a string.

Is this intentional, or should it be like below?
```rust
pub fn format_ether<T: Into<ParseUnits>>(amount: T) -> Result<String, ConversionError> {
    format_units(amount, "ether")
}
```

**Comment by culda** [comment]
+1
I can't work out how to print a decimal representation of ETH, like 0.1

**Comment by culda** [comment]
I mean `format_units(x, 18)` works, but would expect it to be equivalent to `format_ether(x)`

**Comment by AndoroidX** [comment]
Fixed with #2435 
@halo3mic you can close the issue now.
